### ISSUE TITLE
Request Spot Check for Rated Disabilities

### ISSUE THREAD

**Issue by jason-gcio** [department-of-veterans-affairs/va.gov-team#3521]
## Product Name

**Rated Disabilities**
Staging link: https://staging.va.gov/disability/view-disability-rating/rating/
User: vets.gov.user+228@gmail.com
Usual password

## Checkpoint Stage

- [ ] **Design-Development Intent Checkpoint** (Review design to discuss approach to take with development, identify what components and semantic HTML would be appropriate, identify any accessibility issues and how to address) - This would happen as designs are passed to development, before development begins.

- [x] **Spot Check Review Checkpoint** (Review and follow the steps in the [508 Checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-checklist.md)) - This happens well before launch, so that the 508 office can review and any issues addressed *before* launch.

## Points of Contact

Jason Wolf (Slack: @Jason, GitHub: @jason-gcio)   
Jim Adams (Slack: @Jim Adams, GitHub: @sporeboy)  

## Preferred times

Is this done asynchronously or should we find a time to meet?

## Next steps

1. Accessibility specialist will create an epic in Zenhub to capture issues.

2. Team will coordinate another review after work is updated by @-comment in Zenhub in the Github issue.

3. Once all issues are addressed, the ticket is closed with a final comment, “All issues are addressed”, @-comment developer who opened issue, and close ticket/issue.

**Comment by 1Copenut** [comment]
Thanks @jason-gcio . I followed up with @jenstrickland and confirmed she'll be running this spot check. I'm always available for consult or questions as needed.

**Comment by jenstrickland** [comment]
Findings will be collected in this epic: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/3577

**Comment by jenstrickland** [comment]
@jason-gcio I'm trying to access the link [https://staging.va.gov/disability/check-disability-rating/rating/](https://staging.va.gov/disability/check-disability-rating/rating/) but getting an error using the login provided. Perhaps we should schedule a quick call to review accessing the page?

**Comment by jason-gcio** [comment]
@jenstrickland Sure we can...are you able to see it at this url?
https://staging.va.gov/disability/view-disability-rating/rating/

**Comment by jenstrickland** [comment]
@jason-gcio Thank you, it's working now. Jim said there was an issue yesterday, so maybe it's resolved now.

**Comment by jenstrickland** [comment]
Updated url to test: https://staging.va.gov/disability/view-disability-rating/rating/

**Comment by jenstrickland** [comment]
Spot check complete, and collected in epic https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/3577
